osbuild/buildroot.py

```diff
@@ -271,6 +271,11 @@ def run(self, argv, monitor, timeout=None, binds=None, readonly_binds=None, extr
             api_path = "/run/osbuild/api/" + api.endpoint
```

Comment:
(commit message of "buildroot: explicitly bind mount runner ")

`relaying on the assumption` Tztztz, why would you _relay_ an assumption? Such a rude habit.